selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/DefaultAndroidEmulatorTests.java

```diff
@@ -23,15 +23,19 @@
 import org.apache.commons.io.FileUtils;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.Ignore;
 import org.openqa.selenium.Dimension;
 
 public class DefaultAndroidEmulatorTests {
+
+  @Ignore("Fail. Requires list of available avds")
   @Test
   public void shouldBeAbleToListAvds() throws Exception {
     List<AndroidEmulator> avds = DefaultAndroidEmulator.listAvailableAvds();
     Assert.assertFalse("Expecting list of avds not to be empty", avds.isEmpty());
   }
 
+  @Ignore("Fail. Requires emulator to exist on the system running the test.")
```

Comment:
are we assuming a system that is running these tests does not have any emulators? or can't create one?